reqs/management/commands/import_reqs.py

```diff
@@ -251,6 +271,7 @@ def add(self, row):
             req_id=req_id, defaults=params)
         self.connections.extend(self.keywords.connections(row, req.pk))
         self.req_ids.add(req_id)
+        self.last_id = req_id
```

Comment:
It'd be nice if all of the logic around `last_id` were maintained in one place -- does it make sense to set it within `fix_excel_decimals`?